### ISSUE TITLE
The possible deprecation for the `vlbu.v` and `vsb.v`

### ISSUE THREAD

**Issue by vickiegpt** [gsauthof/riscv#2]
For latest build of v extension 1.0.x, we have
```bash
~/project/chocopy/test/acc/riscv on  master [?]
❯ riscv64-unknown-elf-as -march=rv64gcv ./bcd2ascii.s
./bcd2ascii.s: Assembler messages:
./bcd2ascii.s:63: Error: unrecognized opcode `vlbu.v v16,(a1)'
./bcd2ascii.s:94: Error: unrecognized opcode `vsb.v v24,(a0)'
```

So, it should be replaced with `vlw.v` and `vsw.v`. You could inspect my implementation of V Extension [codegen](http://s3l.shanghaitech.edu.cn:8081/yangyw/chocopy/blob/master/include/ir-optimizer/Module.hpp) or [LLVM's](https://github.com/llvm/llvm-project/blob/68bc6d7cae6d5b8679fcf84c73c114409b6cb469/llvm/lib/Target/RISCV/RISCVInstrFormatsV.td)

**Comment by shpwyjymjr** [comment]
I have the same problem and I replace the vlbu.v and vsb.u with vlw.v and vsw.v, while assembling, the same error.

bcd2ascii.s: Assembler messages:

bcd2ascii.s:63: Error: unrecognized opcode `vlw.v v16,(a1)'

bcd2ascii.s:94: Error: unrecognized opcode `vsw.v v24,(a0)'

riscv toolchain branch:master / rvv-next
gcc branch: riscv-gcc-rvv-next
binutils branch: riscv-binutils-2.38

how can i deal with the problem

**Comment by vickiegpt** [comment]
Have you tried the compiler option [here](https://godbolt.org/z/4GE3sdnTv)

**Comment by shpwyjymjr** [comment]
I use riscv-gnu-toolchain, not llvm.
The code is bcd2ascii.s.
After assembling, it shows:
$RVV_NEXT/bin/riscv64-unknown-elf-as -march=rv64gcv1p0 -o bcd2ascii.o bcd2ascii.s
bcd2ascii.s: Assembler messages:
bcd2ascii.s:63: Error: unrecognized opcode `vlw.v v16,(a1)'
bcd2ascii.s:94: Error: unrecognized opcode `vsw.v v24,(a0)'

**Comment by Gandalfthegreybeard** [comment]
I'll guess that the assembler is built from a version of binutils that no longer supports those opcodes, as the RVV spec was finalized at 1.0. You could try a T-Head toolchain as they shipped hardware based on the 0.7 spec. Look for something around binutils 2.32?
